packages/xo-server/src/api/vtpm.mjs

```diff
@@ -0,0 +1,27 @@
+export function create({ vm }) {
+  return this.getXapi(vm).VTPM_create({ VM: vm._xapiRef })
+}
+
+create.description = 'create a VTPM'
+
+create.params = {
+  id: { type: 'string' },
+}
+
+create.resolve = {
+  vm: ['id', 'VM', 'administrate'],
+}
+
+export function destroy({ vtpm }) {
+  return this.getXapi(vtpm).call('VTPM.destroy', vtpm._xapiRef)
```

Comment:
What should this method return?

Reply:
`VTPM.destroy` returns `void`.

Reply:
I believe it returns `''`, am I wrong?

If you don't want to return anything, use `async`/`await` instead.

Reply:
Ho, you're right. I was using `xo-cli` to call `vtpm.destroy`, and the output was "nothing".
But if I make the call with another source it returns an empty string.